src/components/RegistrationEmail.vue

```diff
@@ -0,0 +1,125 @@
+<!--
+  - @copyright Copyright (c) 2022 Carl Schwan <carl@carlschwan.eu>
+  -
+  - @author Carl Schwan <carl@carlschwan.eu>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+<template>
+	<div class="guest-box">
+		<form action="" method="post">
+			<fieldset>
+				<div v-if="message !== ''" class="notecard error">
+					{{ message }}
+				</div>
+
+				<p class="email">
+					<input id="email"
+						type="email"
+						name="email"
+						class="email__field"
+						:placeholder="emailPlaceholder"
+						required
+						autofocus>
+					<label v-if="emailIsOptional" for="email" class="infield">{{ t('registration', 'Email (optional)') }}</label>
+					<label v-else for="email" class="infield">{{ t('registration', 'Email') }}</label>
+					<img class="svg email__icon" :src="emailIconPath" alt="">
+				</p>
+
+				<div id="terms_of_service" />
+
+				<input type="hidden" name="requesttoken" :value="requesttoken">
+				<Button id="submit"
+					native-type="submit"
+					type="primary"
+					:wide="true">
+					{{ submitValue }}
+				</Button>
+
+				<a id="lost-password-back" :href="loginFormLink">
+					{{ t('registration', 'Back to login') }}
```

Comment:
guest.css link color is black, so this is not visible in dark mode, but I guess it will be fixed via the server PR
![Bildschirmfoto vom 2022-07-22 13-37-53](https://user-images.githubusercontent.com/213943/180431550-25aeb161-16c9-465d-b06c-4ac85d08e226.png)

Reply:
fixed